test/exec-specs.js

```diff
@@ -19,7 +17,7 @@ describe('exec', function () {
     let {stdout, stderr, code} = await exec(cmd, args);
     stdout.should.contain('exec-specs.js');
     stderr.should.equal('');
-    code.should.equal(0);
+    should.equal(code, 0);
```

Comment:
this needs to change because `code` is `number|null`, and if it is the latter, `code.should.equal(0)` would throw an exception, since `null` is not an object and certainly not one having a `should` property.